velox/functions/sparksql/aggregates/FirstLastAggregate.cpp

```diff
@@ -51,6 +51,10 @@ class FirstLastAggregateBase
     return sizeof(TAccumulator);
   }
 
+  int32_t accumulatorAlignmentSize() const override {
```

Comment:
This is not required since this is defined in the base class?

Reply:
It's needed by partial specialization of accumulatorAlignmentSize, actually I follow this usage from MinMaxAggregates.cpp#L70